Duplicati/Library/Main/Database/Database schema/Schema.sql

```diff
@@ -82,6 +82,7 @@ CREATE TABLE "FilesetEntry" (
 /* Improved reverse lookup for joining Fileset and File table */
 CREATE INDEX "FilesetentryFileIdIndex" on "FilesetEntry" ("FileID");
 CREATE INDEX "nn_FilesetentryFile" on FilesetEntry ("FilesetID","FileID");
+CREATE INDEX "FilesetEntryFilesetID" ON "FilesetEntry" ("FilesetID")

```

Comment:
The line above has an index on the same leading column.  The cost of maintaining this index will be higher shan using the original one as it will skipscan.

Is there a specific reason for this index or crn it be removed?